src/pseudopeople/configuration/generator.py

```diff
@@ -76,14 +76,20 @@ def get_configuration(user_configuration: Union[Path, str, Dict] = None) -> Conf
     :return: a ConfigTree object of the noising configuration
     """
 
-    noising_configuration = _generate_default_configuration()
+    if type(user_configuration) == str:
+        if user_configuration.lower() == Keys.NO_NOISE:
```

Comment:
You are right there is a gap in the logic. I did want a one liner like you suggest but if user_configuration is None it does not have a .lower() method.

Reply:
Boolean operators short circuit. This means that it doesn't evaluate the entire expression if it already knows what it will evaluate to. Let me give you an example. 

Say you have the if statement `if A and B`, then this will be true if and only if both `A` and `B` are true. Since it reads from left to right, the evaluator will first determine the value of `A`, and then that of `B`. However, if `A` is false, there is no need to evaluate `B`, since no matter what it is, `A and B` will be false. 

There is an equivalent short-circuiting that happens with OR (i.e. if `A` is true in the expression `if A or B` then `B` won't be evaluated). 

This is what python does, so you don't need to worry about `user_configuration` potentially being `None` since `user_configuration.lower()` will only get called if it is a string.